### PR TITLE
changeable namespace

### PR DESCRIPTION
**Pull request by RoiGlinik**

### CODE REVIEW

**Comment by aantn** [comment]
Re-assigning to @arikalon1!
I unfortunately don't have the time right now

**Comment by RoiGlinik** [comment]
its not a high priority. also i forgot a testing file there i will remote it

File: helm/robusta/kubewatch.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-namespace: ""
```
Comment (arikalon1) [inline helm/robusta/kubewatch.yaml:1]: Why did you remove it?
What was the purpose of that?

File: helm/robusta/templates/auth-config.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: robusta-auth-config-secret
+  namespace: {{ .Release.Namespace }}
```
Comment (arikalon1) [inline helm/robusta/templates/auth-config.yaml:6]: Does .Release.Namespace always has a value?
It's either the current namespace, or the one you pass to the helm command?

File: helm/robusta/kubewatch.yaml (hunk 1 above)
Comment (RoiGlinik) [reply helm/robusta/kubewatch.yaml:1]: i moved it to the kubewatch-configmap.yaml file so i can use the release value, the content of the file was used for the config map.

File: helm/robusta/templates/auth-config.yaml (hunk 2 above)
Comment (RoiGlinik) [reply helm/robusta/templates/auth-config.yaml:6]: from my test its set to default when i dont use  something.
